btb_phylo.py

```diff
@@ -378,13 +381,14 @@ def main():
     multi_fasta_path = os.path.join(results_path, "multi_fasta.fas")
     snp_sites_outpath = os.path.join(results_path, "snps.fas")
     snp_dists_outpath = os.path.join(results_path, "snp_matrix.tab")
+    consensus_downloads = '/mnt/fsx-017/phyloConsensus/'
```

Comment:
Could make this more general: the `consensus_downloads` could go somewhere in results path, and then when running for standard VB opperation we set the results path to `fsx-017`.